content/faq/voice/text-to-speech-android-tts/index.fr.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Synthèse vocale sur Android
+description: Guide pour utiliser la synthèse vocale sur Android
+aliases: ["/fr/faq/synthèse-vocale-ttsw-sur-android"]
```

Comment:
By default, the url is formed from the file/directory name: /fr/faq/text-to-speech-android-tts/

If you need to override it with a custom slug, use
`slug: synthèse-vocale-ttsw-sur-android`

`alias` keyword is used for redirects from an older existing address. The address `/fr/faq/synthèse-vocale-ttsw-sur-android` doesn't exist yet, so there is no need to alias it.

It's documented here: https://www.getzola.org/documentation/content/page/#path-from-frontmatter